Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM alpine:3.5
 
+RUN echo 'hosts: files dns' >> /etc/nsswitch.conf
```

Comment:
In-fact this file doesn't even exist @balamurugana inside the container. It is meant for only one purpose.